portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime; 
+public class Comment {
+    
+    private long id;
+    private String name;
+    private String content;
+    private long timestamp;
+
+    public Comment(long id, String name, String content, long timestamp) {
+        this.id = id;
+        this.name = name;
+        this.content = content;
+        this.timestamp = timestamp;
```

Comment:
Optional: I think Java has a native time time, LocalDateTime. Using a native time type could make dealing with timezones and type conversions a little easier.

Reply:
I totally agree. Let me change that.